R/sass.R

```diff
@@ -96,11 +96,19 @@ sass <- function(input = NULL, options = sass_options(), output = NULL,
   }
 
   css <- as_html(css, "css")
+  deps <- html_dependencies(input)
+  deps <- htmltools::resolveDependencies(deps)
```

Comment:
Either don't resolve dependencies, or, pass `FALSE` for `resolvePackageDir` argument